Description: Always use pari_init_stack
 This is required in order to build against PARI 2.9.
Author: Bill Allombert <ballombe@debian.org>
Bug-Debian: http://bugs.debian.org/841738

--- a/src/Lcommandline.cc
+++ b/src/Lcommandline.cc
@@ -473,7 +473,11 @@
 
 #ifdef INCLUDE_PARI
         if(do_elliptic_curve){
+#if PARI_VERSION_CODE >= PARI_VERSION(2,8,0)
+             paristack_resize((size_t)N_terms*16 + 1000000);
+#else
              pari_init_stack((size_t)N_terms*16 + 1000000, top-bot); // Allocate bigger PARI stack
+#endif
              if (my_verbose>0) cout << "Will precompute " << N_terms << " elliptic L-function dirichlet coefficients..." << endl;
              initialize_new_L(a1,a2,a3,a4,a6,N_terms); 
         }
